packages/frameworks-solid-start/.gitignore

```diff
@@ -2,6 +2,7 @@ node_modules
 dist
 **/*.d.ts
 **/*.js
+**/*.jsx
```

Comment:
Are we shipping jsx? Will that work out of the box with SolidStart, or need an extra transpilation step?

Reply:
Ye it works i already set the jsxImportSource to solid, and tried it. Seems to work fine

Reply:
https://github.com/nextauthjs/next-auth/pull/6248/commits/5eea474c74037e1545f2f43fd10b4fa4a14d77ac